charts/aws-vpc-cni/values.yaml

```diff
@@ -46,6 +46,7 @@ nodeAgent:
   networkPolicyAgentLogFileLocation: "/var/log/aws-routed-eni/network-policy-agent.log"
   enableIpv6: "false"
   metricsBindAddr: "8162"
+  metricsBindPort: "8162"
```

Comment:
This should not have made it to the original PR. I am not sure why this was allowed.

Reply:
yeah this looks weird...
it's added by this commit: https://github.com/aws/amazon-vpc-cni-k8s/commit/28c99c9443d2c3a3408cf9d1d67f7e395dd92ee8

Reply:
picked up the fix in master